features/step_definitions/customer_booking_request_steps.rb

```diff
@@ -0,0 +1,31 @@
+Given(/^a location enabled for online booking$/) do
+  Locations.online_booking_location_uids = %w(ac7112c3-e3cf-45cd-a8ff-9ba827b8e7ef)
```

Comment:
This alters the state of the system and could affect future tests. What are your thoughts on using the following instead
```
allow(Locations).to receive(:online_booking?).with('ac7112c3-e3cf-45cd-a8ff-9ba827b8e7ef').and_return(true)
```